channeldb/channel.go

```diff
@@ -1795,7 +1795,7 @@ func syncNewChannel(tx kvdb.RwTx, c *OpenChannel, addrs []net.Addr) error {
 // occur. This method its to be called when we revoke our prior commitment
 // state.
 //
-// A map is returned of all the htlc resolutions that were locked in in this
```

Comment:
was this automatically detected by the grammar check you were referring to?

Reply:
yes! It told me that there were to `in`s😂